tpm2_auth.c

```diff
@@ -40,7 +40,7 @@ u8 *tpm2_null_auth(struct tpmbuff *b)
 	u32 *handle;
 	u8 *auth = (u8 *)tpmb_put(b, NULL_AUTH_SIZE);
 	if (auth == NULL)
```

Comment:
You will probably get nit picks on annoying things like "this should be if (!auth)". Also just return NULL here.

Reply:
ack